romancal/skycell/skymap.py

```diff
@@ -48,13 +44,12 @@ def __init__(self, index: int | None, skymap: "SkyMap" = None):
         skymap: SkyMap
             sky map instance (defaults to global SKYMAP)
         """
-
-        if skymap is None:
-            skymap = SKYMAP
-
-        self._index = index
-        self._skymap = skymap
-        self._data = None
+        warnings.warn(
+            "The SkyCell object is deprecated and will be removed in a future release. Use the SkyCells object instead.",
+            category=DeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
Is this class documented (I can't find it in rtd)? If not, do we need a deprecation? If there is no known downstream usage I'm in favor of removing it without a deprecation.

Reply:
Oh, docs! I knew I forgot something 🤦 

I wasn't sure if anyone has used it so didn't want to rug-pull them, but you're right it's been out for so short a time that I doubt there are any other usages other than here